external/GravitationalEffectiveSource/kerr-circular.c

```diff
@@ -0,0 +1,1801 @@
+/*******************************************************************************
+ * Copyright (C) 2012-2013 Barry Wardell
+ *******************************************************************************/
+
+#include <math.h>
+#include "effsource.h"
```

Comment:
Have you tried compiling this?

Reply:
good catch, fixed and squashed in the rename to `effsource_gr.h` (this got lost when I pulled in the most recent revision of the file from the source repo).